src/api/js/app.js

```diff
@@ -0,0 +1,245 @@
+'use strict';
+
+const express = require('express');
+let serviceLibrary = process.env.NODE_WEB_SERVICE || 'rest.js';
+const service = require('./lib/'+serviceLibrary);
+var morgan = require('morgan');
```

Comment:
should avoid using `var` keyword, use `let` instead.

Reply:
Extracted into issue #53